index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Product Landing Page</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      Explore
+      Start a project
+      <img src="logo.png">
+      <input type="text" placeholder="Search">
+      Log in
+    </nav>
+    <h1>The Minimalist’s Wall Calendar | Visualize Your Year</h1>
+    <p>Minimalist, forgiving, and beautiful. A wall calendar designed to make 2021 feel like a breath of fresh air.</p>
+  </header>
+  <main>
+    <figure>
+      <img src="hero.png">
+      <figcaption>Project we Love | Product Design | Cambridge, MA</figcaption>
+    </figure>
+    <aside>
+      <hr>
+      <p>
+        $13,757<br>
+        pledged of $1,200 goal
+      </p>
+      <p>
+        215 <br>
+        backers
+      </p>
+      <p>
+        19 <br>
+        days to go
+      </p>
+      <input type="button" value="Back this Project"><br>
+      <input type="button" value="Remind me">
```

Comment:
`input` elements are designed to be associated with Form elements more than they are meant to signify actions that a user can take.

In general, there are three kinds of interactive elements on the page that can all be made to look like buttons:

element | usage
---- | ----
`<button>` | a clickable button, used to submit forms or anywhere in a document for accessible, standard button functionality.
`<input type="button">` | A push button associated with a form that has no default behavior displaying the value of the value attribute, empty by default.
`<a>` | a hyperlink to web pages, files, email addresses, locations in the same page, or anything else a URL can address. Content within each <a> should indicate the link's destination.

As a quick reminder: HTML is about meaning, not design, you can make almost any HTML element look like whatever you want.

Given the above table, generally, you'll want to use `<a>` tags if the button is meant to take you somewhere else (like another page), you'll want to use a `<button>` tag if the button takes an action for you (like saving the content of a page or adding a "like" to a post), and an `<input type="button">` to take some action in the context of a form _only_.

Given that, the more appropriate element tag here should likely be a `<button>` or an `<a>` tag, depending on whether you think it should take action directly or navigate the user somewhere else.